webapp/shop/api/credly/api.py

```diff
@@ -17,9 +17,9 @@ def __init__(
         self.session = session
         self.auth_token = auth_token
         self.badge_template_dict = {
-            4190: "3d3a19a1-324c-48fa-aa55-60f66c0dcca6",
```

Comment:
I assume here you are changing the keys to match prod keys.
Are you sure you want to use prod API on all environments? If no, these values should be in the env file and konf.